src/Plugin.ts

```diff
@@ -77,6 +79,10 @@ export class Plugin {
   }
 
   public async recordHar(options: RecordOptions): Promise<void> {
+    if (this.browser && !this.isSupportedBrowser(this.browser)) {
```

Comment:
```suggestion
    if (!this.browser || !this.isSupportedBrowser(this.browser)) {
```

ditto elsewhere

Reply:
This doesn't make much sense, as this.browser is always defined according to the logic and type declaration